django/forms/models.py

```diff
@@ -462,6 +462,29 @@ def save(self, commit=True):
                              fail_message, commit, self._meta.exclude,
                              construct=False)
 
+    def save(self, commit=True):
+        """
+        Public version of ``ModelForm._save()``
+
+        Method broken down into public and private versions in order to avoid loops
+        when overriding.
+        """
+        return self._save(commit)
+
+    def get_updated_model(self):
+        """
+        Saves this ``form``'s cleaned_data into model instance ``self.instance``.
+
+        Returns ``instance``.
+
+        Changes to ``instance`` are never saved to database.
+
+        This is a utility method so newcomers following tutorials never have to write
+        something that looks like "save but not save" (``form.save(commit=False)``)
+        just to acquire an updated instance of a model.
+        """
+        return self._save(commit=False)
+
     save.alters_data = True
```

Comment:
This line is related to the `save` method, and should sit beneath it, rather than the new `get_updated_model` method.